packages/prisma/index.ts

```diff
@@ -21,7 +22,6 @@ if (process.env.NODE_ENV !== "production") {
 }
 // If any changed on middleware server restart is required
 bookingReferenceMiddleware(prisma);
-eventTypeDescriptionParseAndSanitizeMiddleware(prisma);
```

Comment:
@zomars & @emrysal I removed that middleware now as `descriptionAsSafeHTML` is coming from all parts of my code already. Any objections to this?

Reply:
@zomars what do you think about removing the middleware for sanitizing the event type description?